plugins/generator-1.14.4/forge-1.14.4/triggers/player_xp_change.java.ftl

```diff
@@ -0,0 +1,20 @@
+@SubscribeEvent public void onPlayerXPChange(PlayerXpEvent.XpChange event) {
+	if (event != null && event.getEntity() != null) {
+		Entity entity = event.getEntity();
+		double i=entity.posX;
+        double j=entity.posY;
+        double k=entity.posZ;
+		int amount = event.getAmount();
+		World world = entity.world;
+

```

Comment:
There is an empty line here and the `double j` and `double k` lines have also a different indentation.